package.json

```diff
@@ -738,7 +738,7 @@
         },
         "powershell.developer.featureFlags": {
           "type": "array",
-          "default": [],
+          "default": null,
```

Comment:
Does this actually do anything?  I was under the impression that this was just normal JSON schema for intellisense in the settings.  I absolutely be wrong though.

For example, here's a default from [terminal-input](https://github.com/SeeminglyScience/terminal-input/blob/6e1ab94e8f6038f387c11d2b5630dcf96afa5418/package.json#L28-L43)

```json
		"configuration": [{
			"title": "Terminal Input",
			"properties": {
				"terminalInput.customInputs": {
					"title": "Custom Inputs",
					"description": "Strings that can be used by the sendCustomInput command to send raw input to the active terminal.",
					"type": "object",
					"default": { "heartEmojiWithAlt": "\u001b\u2665" },
					"additionalProperties": [{
						"type": "string",
						"title": "Input",
						"description": "Text to send to the terminal as raw input."
					}]
				}
			}
		}]
```

But that doesn't actually create any defaults, it comes back as `undefined`.

Reply:
without setting this to `null`, the default becomes `[]` and thus [this line](https://github.com/PowerShell/vscode-powershell/blob/master/src/settings.ts#L211) behaves differently. I can't modify that line because a user should be able to opt out of PSRL on Windows by doing this in their settings:

```
powershell.developer.featureFlags = []
```

for context... in node:
```
PS > node
> if([]){ "asdf" }
'asdf'
> if(null){ "asdf" }
undefined
>
```

Reply:
in otherwords... I tested it, doesn't work that way 😊